packages/luigis-box/src/Model/Article/ArticlesSearchResultsProvider.php

```diff
@@ -33,9 +35,11 @@ public function getArticlesSearchResults(
         Argument $argument,
     ): Promise|array {
         return $this->luigisBoxBatchLoader->load(
-            new LuigisBoxBatchLoadData(
+            $this->luigisBoxBatchLoadDataFactory->create(
                 'article',
```

Comment:
why do not you use constants from `LuigisBoxClient` here (and in other places where the `create` method is called)?